recipes/access/meta.yaml

```
@@ -0,0 +1,55 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "access" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e3f8c939f8d7c8abae6b7b4b12e02d6f6f357495d42d047c189d009fde379ba5
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3.5
+    - scipy >=0.11
+    - numpy >=1.3
+    - pandas >=0.23.4
+    - Sphinx ==2.4.3
```

```suggestion
    - Sphinx ==2.4.3
```
Sphinx is probably only required for doc generation. Can perhaps be dropped?